packages/gatsby-theme-carbon/src/components/FeatureCard/feature-card.scss

```diff
@@ -89,4 +89,10 @@
 .#{$prefix}--feature-card .gatsby-resp-image-wrapper {
   margin-left: 0;
   margin-right: 0;
+  @include carbon--breakpoint('lg') {
+    max-width: 100% !important;
```

Comment:
does it not work without the `!important`?

Reply:
I wish it did. But it was the only way I could get it to take precedence over gatsby's insistence that the max-width be hardcoded at 1152px.